src/status.js

```diff
@@ -1,65 +1,97 @@
-class Check {
-  todoList(todoList) {
-    const list = document.querySelector('#todo-data');
-    if (this.getItems() === null) {
-      todoList.forEach((task, i) => {
-        if (task.completed === true) {
-          const li = document.createElement('li');
-          const text = `<div class="list-container"> <input class='check-input' type='checkbox' id='${i}' aria-label='...' checked>
-          <p class="todo-text">${task.itemText}</p><a class="del-menu" href="#"><i class="fas fa-ellipsis-v"></i></a></div>`;
-          li.classList.add('list-item');
-          li.innerHTML = text;
-          list.appendChild(li);
-        } else {
-          const li = document.createElement('li');
-          const text = `<div class="list-container"> <input class='check-input' type='checkbox' id='${i}' aria-label='...'>
-            <p class="todo-text">${task.itemText}</p><a class="del-menu" href="#"><i class="fas fa-ellipsis-v"></i></a></div>`;
-          li.classList.add('list-item');
-          li.innerHTML = text;
-          list.appendChild(li);
-        }
+class Status {
+  constructor(form, todoTask, mainList, list, data, completed) {
+    this.form = form;
+    this.todoTask = todoTask;
+    this.mainList = mainList;
+    this.list = list;
+    this.data = data;
+    this.completed = completed;
```

Comment:
- A todo item should not be false by default. Please assign a default false value to the propery. `this.completed=false`

Reply:
Hello, Thank you for such a prompt review.

I'll urge you to double-check this, please, the text that you have highlighted should've been used as an argument but I am not using that. So, I am going to remove that now. 
Plus it clearly says in the requirements that the property should be set to `false` and the `index` should be the length of node items. I have just manipulated data to make it more dynamic.

![Screenshot 2021-10-30 at 9 48 10 PM](https://user-images.githubusercontent.com/57455864/139541914-b86cb36a-cb4d-4c33-a46c-fbf34456cf76.png)